core/src/avm1.rs

```diff
@@ -645,10 +662,11 @@ impl<'gc> Avm1<'gc> {
             args.push(self.pop()?);
         }
 
-        let target_fn = self
-            .current_stack_frame_mut()
-            .unwrap()
-            .resolve(fn_name.as_string()?);
+        let target_fn = self.stack_frames.last().unwrap().clone().read().resolve(
```

Comment:
```
let stack_frame = self.current_stack_frame().unwrap().read().resolve(
```

Reply:
Uh... Is this correct? `target_fn` is the thing we actually want to call...